model/build-processor/src/main/java/org/keycloak/models/map/processor/GenerateEntityImplementationsProcessor.java

```diff
@@ -531,25 +531,25 @@ public void generate(TypeElement e) throws IOException {
                 pw.println("        return this.entityFieldDelegate;");
                 pw.println("    }");
 
+                pw.println("    @Override public boolean isUpdated() {");
+                pw.println("        return entityFieldDelegate.isUpdated();");
+                pw.println("    }");
+
+                pw.println("    @Override public void clearUpdatedFlag() {");
+                pw.println("        entityFieldDelegate.clearUpdatedFlag();");
+                pw.println("    }");
+
                 getAllAbstractMethods(e)
                   .forEach(ee -> {
-                      String originalField = m2field.get(ee);
-                      if (originalField == null) {
-                          return;
-                      }
-                      TypeMirror fieldType = determineFieldType(originalField, methodsPerAttribute.get(originalField));
-                      String field = fieldsClassName + "." + toEnumConstant(originalField);
+                    String originalField = m2field.get(ee);
+                    if (originalField == null) {
+                        return;
+                    }
+                    TypeMirror fieldType = determineFieldType(originalField, methodsPerAttribute.get(originalField));
+                    String field = fieldsClassName + "." + toEnumConstant(originalField);
```

Comment:
Are those indentation changes intentional?

Reply:
nope, thanks for spotting it